src/client/components/fields/DateField.tsx

```diff
@@ -8,46 +8,64 @@ import {
   FormHelperText,
   FormErrorMessage,
   Input,
+  InputGroup,
+  InputRightElement,
 } from '@chakra-ui/react'
 
 import 'react-datepicker/dist/react-datepicker.css'
 import '../../styles/date-picker.css'
 
 import { Field } from '../../../types/checker'
+import { BiCalendar } from 'react-icons/bi'
+
+interface DatePickerInputProps {
+  value: string
+  onClick: (event: React.MouseEvent<HTMLInputElement, MouseEvent>) => void
+}
+
+const DatePickerInput: FC<DatePickerInputProps> = ({ value, onClick }) => (
+  <InputGroup>
+    <Input
+      type="text"
+      placeholder="DD/MM/YYYY"
+      className="react-datepicker-ignore-onclickoutside"
+      value={value}
+      onClick={onClick}
```

Comment:
```suggestion
      onClick={onClick}
      readOnly
```

Add to suppress warning:
```
You provided a `value` prop to a form field without an `onChange` ... Otherwise, set either `onChange` or `readOnly`
```